src/Xamarin.Android.Build.Tasks/Tasks/D8.cs

```diff
@@ -124,5 +124,11 @@ protected virtual CommandLineBuilder GetCommandLineBuilder ()
 
 			return cmd;
 		}
+
+		protected override void LogEventsFromTextOutput (string singleLine, MessageImportance messageImportance)
```

Comment:
It might be worth putting a comment why we don't call `base.LogEventsFromTextOutput()`.